README.md

```diff
@@ -1,19 +1,26 @@
-# emacs-slack
+<p align="center"><img src="assets/slack-logo.svg" width=300 height=126/></p>
```

Comment:
Is this image expected to be broken in GitHub preview?

Reply:
I've never used GitHub preview to see an image preview, but I can see it in README. I can change my image if necessary.

Reply:
Cool!

Reply:
Thanks!
I want to separate media files from source code. 
Could you create pull request to [assets branch](https://github.com/yuya373/emacs-slack/tree/assets) including svg in assets folder, and then remove svg from this pull request?

Reply:
Of course, I'll do that to you right now.

Reply:
img's src attribute will be `https://github.com/yuya373/emacs-slack/raw/assets/assets/slack-logo.svg`

Reply:
You're right. Excuse me for this awkwardness.

Reply:
need `?sanitize=true`  to display svgs 😢 
`https://raw.githubusercontent.com/yuya373/emacs-slack/assets/assets/slack-logo.svg?sanitize=true`

https://github.com/github/markup/issues/556#issuecomment-399336652

Reply:
This should be settled now. Sorry about the rehearsal.

Reply:
🎉